src/target/arc_cmd.c

```diff
@@ -1001,6 +1001,9 @@ const struct command_registration arc_cache_group_handlers[] = {
 		.usage = "",
 		.chain = arc_l2_cache_group_handlers,
 	},
+	{
```

Comment:
This should be placed under the `arc_monitor_command_handlers` in order to be consistent with the implementation on other targets, currently arc commands must be used like `arc cache rtt`, which is incorrect